cmd/dbufstat/dbufstat

```diff
@@ -343,7 +343,7 @@ def get_compstring(c):
                     "ZIO_COMPRESS_GZIP_6",  "ZIO_COMPRESS_GZIP_7",
                     "ZIO_COMPRESS_GZIP_8",  "ZIO_COMPRESS_GZIP_9",
                     "ZIO_COMPRESS_ZLE",     "ZIO_COMPRESS_LZ4",
-                    "ZIO_COMPRESS_FUNCTION"]
+                    "ZIO_COMPRESS_ZSTD",    "ZIO_COMPRESS_FUNCTION"]
```

Comment:
Shouldn't new stuff get added at the end?

Reply:
no. ZIO_COMPRESS_FUNCTION is the end definition of the array and must stay at the end

Reply:
oops. my fault....